### PR TITLE
Add README & posting rules and some basic categories

### PR DESCRIPTION
**Pull request by onimsha**
Follow up with #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+
+## Quick Links
+
+:point_right: [DevOps Jobs](DevOps/README.md)  
+:point_right: [QA Jobs](QA-Engineer/README.md)  
+:point_right: [Software Jobs, BackEnd](Software-Engineer-BackEnd/README.md)  
+:point_right: [Software Jobs, FrontEnd](Software-Engineer-FrontEnd/README.md)  
+:point_right: [Software Jobs, IOS](Software-Engineer-IOS/README.md)  
+:point_right: [Software Jobs, Android](Software-Engineer-Android/README.md)  
+:point_right: [Software Jobs, ReactNative](Software-Engineer-ReactNative/README.md)
```
Comment (ngoduykhanh) [inline README.md]: Should we have ReactNative as a dedicated category rather than apart of front-ends/mobile ones?

Comment (onimsha) [reply README.md]: I dunno. People use ReactNative to build a complete mobile app, similar to Android/IOS, so i thought it should be on a dedicated category. 

@vn-patriot any thought on this ?

Comment (vn-patriot) [reply README.md]: In my opinion, both are correct & insightful enough however I would make it a bit more simpler. So let say we create  2 categories 
- FrontEnd (Web)
- ReactNative (Mobile)

Delete Software Engineer IOS & Software Engineer Android 

However inside React Native mobile we can set some labels like (iOS) / (Android).

Comment (ngoduykhanh) [reply README.md]: Not all people use ReactNative for mobile development, I think it is better to just keep them IOS and Android.
My initial thought was React may dead in the future :-))

Comment (onimsha) [reply README.md]: ok so let's remove this category

Comment (onimsha) [reply README.md]: removed by commit 3c52e5a550cf7dcd6fe3095dd3257e6bcc3632e0

**Review by ngoduykhanh — APPROVED** [review]

**Pull request merged**